config/tools-manual.json

```diff
@@ -168,6 +168,19 @@
                     "categories": ["code-generator", "documentation-generator"]
                 }
             },
+            {
+                "title": "AsyncAPI Modelina",
+                "description": "Modelina is a tool you can use to generate data models to represent the message payloads in a highly customizable manner.",
+                "links": {
+                    "docsUrl": "https://github.com/asyncapi/modelina",
+                    "repoUrl": "https://github.com/asyncapi/modelina"
+                },
+                "filters": {
+                    "language": "TypeScript",
+                    "technology": ["Kotlin", "Java", "TypeScript", "JavaScript", "Rust", "Go", "Dart", "C#", "Python"],
```

Comment:
@jonaslagoni @derberg Are these languages should be added as a part of technology?

Reply:
yeah @jonaslagoni I suggest not to put it in technology as this is not a technology that modelina uses. Just put a full list in description.

also, side question, do you really want to add it manually? don't you want to control it on your own in `modelina` repo through `.asyncapi-tool`? maybe you did not know such option is possible? https://deploy-preview-1265--asyncapi-website.netlify.app/blog/new-asyncapi-tools-page

Reply:
@jonaslagoni pingy